rust/syntect-plugin/manifest.toml

```diff
@@ -6,9 +6,67 @@ activations = ["autorun"]
 commands = []
 
 [[languages]]
-name = "Plain Text"
-extensions = ["txt"]
-scope = "text.plain"
+name = "Assembly x86 (NASM)"
+extensions = ["asm", "inc", "nasm"]
+scope = "source.assembly"
+
+[[languages]]
+name = "Crystal"
+extensions = ["cr"]
+first_line_match = "^#!/.*\\bcrystal"
+scope = "source.crystal"
+
+[[languages]]
+name = "Dart"
```

Comment:
are these languages included in our new repo's packages, or is this from when you were using that other source?

Reply:
Uhm, I think there has been an issue here. I have updated the manifest.toml in the upcoming commit.